README.md

```diff
@@ -21,8 +21,8 @@
 In order to build this project, you need to first clone the repo and then run `make install`:
 
 ```sh
-> git clone https://github.com/ipfs/http-api-docs/http-api-docs
-> cd http-api-docs
+> go get https://github.com/ipfs/http-api-docs/http-api-docs
```

Comment:
Should just be `github.com/...`

Reply:
If we’re making this change, we should also change the text above about “first clone the repo.” It’d also be good to note that you have to have Go installed first.